wazo_confgend/templates/asterisk/extensions/ivr.jinja

```diff
@@ -13,7 +13,8 @@ same  =   n,Goto(t,1)
 {%- endif %}
 
 {% for choice in ivr.choices %}
-exten = {{ choice.exten }},1,Gosub(forward,s,1({{ choice.destination.gosub_args | replace(";", "\;") }}))
+exten = {{ choice.exten }},1,CELGenUserEvent(WAZO_IVR_CHOICE, {"xivo-ivr-{{ ivr.id }}": "{{ choice.exten }}"})
```

Comment:
To follow other [CELGenUserEvent syntax](https://github.com/wazo-platform/xivo-config/blob/master/dialplan/asterisk/extensions_lib_conference.conf#L22), do you mind if we write it like this?
```suggestion
exten = {{ choice.exten }},1,CELGenUserEvent(WAZO_IVR_CHOICE, {"id": {{ ivr.id }},"exten": "{{ choice.exten }}"})
```

Reply:
To make your life easier, is there a guideline somewhere to follow for syntax and other common wazo formats?

Reply:
Unfortunately, no